src/os/bluestore/BlueStore.cc

```diff
@@ -1731,6 +1741,19 @@ void BlueStore::SharedBlobSet::dump(CephContext *cct)
 #undef dout_prefix
 #define dout_prefix *_dout << "bluestore.blob(" << this << ") "
 
+void BlueStore::Blob::dump(Formatter* f) const
+{
+  if (is_spanning()) {
+    f->dump_unsigned("spanning id ", id);
+  }
+  blob.dump(f);
+  if (shared_blob) {
+    f->open_object_section("shared");
+    shared_blob->dump(f);
+    f->close_section();
```

Comment:
fyi you can replace those three lines with ``f->dump_object("shared", *shared_blob)``